src/shared/components/home/site-form.tsx

```diff
@@ -908,6 +909,9 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
 
   handleSiteEnableNsfwChange(i: SiteForm, event: any) {
     i.state.siteForm.enable_nsfw = event.target.checked;
+    if (!event.target.checked) {
+      i.state.siteForm.content_warning = "";
+    }
```

Comment:
I noticed that `enable_nsfw` is actually unused in the backend and removed it (https://github.com/LemmyNet/lemmy/pull/5311). So it should also be removed here.

Reply:
We'll handle that on the next lemmy-js-client types generation. This form field at least just uses a checkbox to show the content warning field.